roles/etcd/templates/etcd.conf.j2

```diff
@@ -1,15 +1,15 @@
 {% macro initial_cluster() -%}
 {% for host in groups[etcd_peers_group] -%}
 {% if loop.last -%}
-{{ host }}={{ etcd_peer_url_scheme }}://{{ hostvars[host]['ansible_' + etcd_interface]['ipv4']['address'] }}:{{ etcd_peer_port }}
+{{ hostvars[host]['ansible_hostname'] }}={{ etcd_peer_url_scheme }}://{{ hostvars[host]['ansible_fqdn'] }}:{{ etcd_peer_port }}
```

Comment:
@detiber I'm curious how you want to handle this one given your previous comment.

Ideally these are the values you would want, but do variables in https://github.com/openshift/openshift-ansible/blob/master/playbooks/common/openshift-etcd/config.yml#L71 propagate to hostvars/facts?